etc/docker/server/rucio.conf.j2

```diff
@@ -30,16 +33,15 @@ CacheRoot /tmp
  SSLVerifyClient optional_no_ca
  SSLVerifyDepth 10
  SSLOptions +StdEnvVars
- 
+
  #AB: SSLv3 disable
  SSLProtocol              all -SSLv2 -SSLv3
  #AB: for Security
  SSLCipherSuite           HIGH:!CAMELLIA:!ADH:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!3DES
-
+{% endif %}
  LogLevel info
  CustomLog logs/access_log combinedrucio
 
- WSGIScriptAlias /auth                   /usr/lib/python2.7/site-packages/rucio/web/rest/authentication.py
```

Comment:
If you remove the authentication endpoint then we need to provide a specific image for this